infrastructure/src/test/java/org/corfudb/infrastructure/CorfuManagementServerTest.java

```diff
@@ -0,0 +1,645 @@
+package org.corfudb.infrastructure;
+
+import com.google.common.util.concurrent.MoreExecutors;
```

Comment:
Can we drop the "Corfu" prefix from the naming convention here, I feel like its superfluous.  Why don't you just replace the tests in ManagementServerTest?

Reply:
I removed the "Corfu" prefix from our tests, but prefer to also leave the old tests there for the time being.